src/component/button.js

```diff
@@ -418,6 +418,7 @@ const copyStyles = StyleSheet.create({
   },
   text: {
     fontFamily: 'OpenSans Light',
+    textAlign: 'center',
```

Comment:
... changing the CopyButton for desktop shouldn't be necessary.

Reply:
fixed!